certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2778 @@
+          "type": "string",
+          "description": "COMPANY DISCRETIONARY DATA",
+          "example": "G"
+        },
+        "entryDescription": {
+          "type": "string",
+          "description": "Description of the entry",
+          "example": "TAXPAYMENT"
+        },
+        "effectiveEntryDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Effective entry date of the transaction",
+          "example": 200130
+        }
+      }
+    },
+    "RTP": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
+      "properties": {
+        "created": {
+          "type": "string",
+          "format": "date",
+          "description": "Transaction creation date",
+          "example": "2022-01-05T12:47:59.000Z"
+        },
+        "debtorClientReference": {
+          "type": "string",
+          "description": "Debtor Client Reference Number",
+          "example": "NOREF"
+        },
+        "debtorBankReference": {
+          "type": "string",
+          "description": "Debtor bank reference number",
+          "example": "20220105042000013P1BRRRR00167874862"
+        },
+        "debtorName": {
+          "type": "string",
+          "description": "Debtor Name",
+          "example": "WESTERN GLAZIERS RETIREMENT PLANDEP OSIT ACCOUNT"
+        },
+        "debtorAccount": {
+          "type": "string",
+          "description": "Debtor Account",
+          "example": 169709692134
+        },
+        "debtorBankID": {
+          "type": "string",
+          "description": "Debtor Bank Identifier",
+          "example": 123000220
+        },
+        "debtorBank": {
+          "type": "string",
+          "description": "Debtor Bank",
+          "example": "US Bank"
+        },
+        "debtorAddress": {
+          "type": "string",
+          "description": "Debtor Address",
+          "example": "PO BOX 4148 PORTLAND 972084148"
+        },
+        "creditorName": {
+          "type": "string",
+          "description": "Creditor Name",
+          "example": "IBEW & United Workers FCU"
+        },
+        "creditorAccount": {
+          "type": "string",
+          "description": "Creditor Account",
+          "example": 323075194
+        },
+        "creditorBankID": {
+          "type": "string",
+          "description": "Creditor Bank Identifer",
+          "example": 244084264
+        },
+        "creditorBank": {
+          "type": "string",
+          "description": "Creditor Bank",
+          "example": "Corporate One Federal Credit Union"
+        },
+        "creditorAddress": {
+          "type": "string",
+          "description": "Creditor Address",
+          "example": "9955 SE Washington St Portland 97216"
+        }
+      }
+    },
+    "Zelle": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",

```

Comment:
@jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".